### ISSUE TITLE
[BUG] version 2.5.0 | started seeing 15-16 popups telling me there were issues

### ISSUE THREAD

**Issue by cobra-999** [DaymareOn/hdtSMP64#145]
The first one read: This tag isn't set in the loading file: log level

It has a OK to select, but then 14-15 others come up and I need to select OK for each of them to run my game.

**Comment by DaymareOn** [comment]
Does this happen each time you start your game, or has it happened only once?

**Comment by cobra-999** [comment]
this happen each time I start my game

**Comment by cobra-999** [comment]
![IMG_20240611_105639](https://github.com/DaymareOn/hdtSMP64/assets/170789657/fce1c587-b8a2-43b3-a04c-dbc29df13075)
![IMG_20240611_105634](https://github.com/DaymareOn/hdtSMP64/assets/170789657/1c9b4e8d-6847-4682-8212-ef27bbbfc87a)
![1718074573](https://github.com/DaymareOn/hdtSMP64/assets/170789657/fb4cf77c-a00d-472b-b162-d872275e7c37)

**Comment by DaymareOn** [comment]
Did you install all the requirements? Please recheck.

**Comment by Chasue** [comment]
Hi, I had the same issue, despite having all requirements installed. But I managed to track down the culprit, so I would like to share my solution:
For me, a mod (Simply Knock) contained a copy of the PapyrusUtil.dll. As I had it lower on my install order, naturally the **older** PapyrusUtil.dll provided by Simply Knock won the file conflict and overwrote the **newer** PapyrusUtil.dll provided by PapyrusUtil SE.

Check your PapyrusUtil in MO2 for any losing conflicts and push whichever mods are overwriting the .dll above PapyrusUtil in your install order to resolve them. Make sure to check whether this causes any undesirable conflicts between other mods after. Alternatively you can probably also simply remove the conflicting .dll itself.

Hope this helps!

![papyrusresolved](https://github.com/DaymareOn/hdtSMP64/assets/172755790/a924b657-aae2-4623-ae26-78032ad9f748)

**Comment by DaymareOn** [comment]
Thank you Chasue!

**Comment by Valadrel** [comment]
In case it helps others, for me the issue was that PapyrusUtil SE - Scripting Utility Functions was still at v3.9. Once I updated to the AE SE v4.6, the popups stopped.

**Comment by 2912635338** [comment]
Sorry, my English is not very good
I also encountered the same problem
Does it need this plugin to function properly?
https://www.nexusmods.com/skyrimspecialedition/mods/13048
Due to the MOD author closing the comment section, I have to register an account here to submit questions

**Comment by 2912635338** [comment]
> Sorry, my English is not very good I also encountered the same problem Does it need this plugin to function properly? https://www.nexusmods.com/skyrimspecialedition/mods/13048 Due to the MOD author closing the comment section, I have to register an account here to submit questions


I don't know what problem I encountered, but my files were configured correctly and read. (This plugin runs normally in my game, except for a bunch of error windows popping up every time I read and save.)
Possible reasons: However, due to some reasons, the game seems unable to detect it
Solution: Ignore or disable error pop ups, but I don't know how to operate or modify the file to disable it. Maybe you can create a version that disables error prompts? Thank you.

**Comment by snottyone** [comment]
> Sorry, my English is not very good I also encountered the same problem Does it need this plugin to function properly? https://www.nexusmods.com/skyrimspecialedition/mods/13048 Due to the MOD author closing the comment section, I have to register an account here to submit questions

yes, it does need that, and it is listed as a requirement on the FSMP nexus page.

**Comment by snottyone** [comment]
> > Sorry, my English is not very good I also encountered the same problem Does it need this plugin to function properly? https://www.nexusmods.com/skyrimspecialedition/mods/13048 Due to the MOD author closing the comment section, I have to register an account here to submit questions
> 
> I don't know what problem I encountered, but my files were configured correctly and read. (This plugin runs normally in my game, except for a bunch of error windows popping up every time I read and save.) Possible reasons: However, due to some reasons, the game seems unable to detect it Solution: Ignore or disable error pop ups, but I don't know how to operate or modify the file to disable it. Maybe you can create a version that disables error prompts? Thank you.
> 
> Addendum: It seems that my game cannot run the above script plugin. Even without it, the action module can still run normally, so the current problem is how to eliminate these annoying pop ups

it runs with ANY skyrim version, if you install the right version.

**Comment by 2912635338** [comment]
> > > Sorry, my English is not very good I also encountered the same problem Does it need this plugin to function properly? https://www.nexusmods.com/skyrimspecialedition/mods/13048 Due to the MOD author closing the comment section, I have to register an account here to submit questions
> > 
> > 
> > I don't know what problem I encountered, but my files were configured correctly and read. (This plugin runs normally in my game, except for a bunch of error windows popping up every time I read and save.) Possible reasons: However, due to some reasons, the game seems unable to detect it Solution: Ignore or disable error pop ups, but I don't know how to operate or modify the file to disable it. Maybe you can create a version that disables error prompts? Thank you.
> > Addendum: It seems that my game cannot run the above script plugin. Even without it, the action module can still run normally, so the current problem is how to eliminate these annoying pop ups
> 
> it runs with ANY skyrim version, if you install the right version.


Okay, this is strange, although I can't be sure about the specific cause of the problem: I tried: 1.50.9-rc1, it's currently working properly
For me, installing the most basic physical support is enough, I didn't use any fancy things

**Comment by z78102** [comment]
我也是碰到這個問題 也是慢慢抓問題點 試過很多方式 最後發現不要使用mod內建的FSMP MCM  獨立安裝MCM 就不會錯報了

**Comment by XHang** [comment]
> 我也是碰到這個問題 也是慢慢抓問題點 試過很多方式 最後發現不要使用mod內建的FSMP MCM 獨立安裝MCM 就不會錯報了

It solved my problem.
For others who have the same issue, DO NOT CHECK THE OPTION FSMP MCM WHILE INSTALL THIS MOD

**Comment by That-Box** [comment]
> 我也是碰到這個問題 也是慢慢抓問題點 試過很多方式 最後發現不要使用mod內建的FSMP MCM 獨立安裝MCM 就不會錯報了

This was a life savor for me in VR. in VR, all these prompts can get you soft-locked in the main menu since it loads into a world. 

For anyone who is too lazy to use google translate;
_I also encountered this problem, and I slowly figured out the problem. After trying many methods, I finally found that you should not use the FSMP MCM built in the mod. Installing the MCM independently will avoid false alarms._

**Comment by tim-elmer** [comment]
For any random individual that encounters this issue via searching, this can also occur if you _forgot_ to install PapyrusUtil for a mod that needs it.
